src/mmrelay/meshtastic_utils.py

```
@@ -474,6 +474,161 @@ class MetadataExecutorDegradedError(RuntimeError):
+    "requires_continuous_health_monitor",
+    "reset_executor_degraded_state",
+    "save_longname",
+    "save_shortname",
+    "send_text_reply",
+    "serial",
+    "serial_port_exists",
+    "shutdown_shared_executors",
+    "sync_name_tables_if_changed",
+    "time",
+    "unsubscribe_meshtastic_callbacks",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)